include/librealsense2/h/rs_option.h

```diff
@@ -175,6 +176,16 @@ extern "C" {
     } rs2_cah_trigger;
     const char* rs2_cah_trigger_to_string( rs2_cah_trigger preset );
 
+    /** \brief values for RS2_OPTION_HOST_PERFORMANCE option. */
+    typedef enum rs2_host_perf_mode
+    {
+        RS2_HOST_PERF_DEFAULT = 0,  /**< no change in settings, use device defaults */
+        RS2_HOST_PERF_LOW = 1,      /**< low performance host mode, if host cannot keep up with workload, this option may improve stability, for example, it sets larger USB transaction granularity, reduces number of transactions and improve performance and stability on relatively weak hosts as compared to the workload */
+        RS2_HOST_PERF_HIGH = 2,     /**< high performance host mode, if host is strong as compared to the work and can handle workload without delay, this option sets smaller USB transactions granularity and as result larger number of transactions and workload on host, but reduces chance in device frame drops */
+        RS2_HOST_PERF_COUNT         /**< Number of enumeration values. Not a valid input: intended to be used in for-loops. */
+    } rs2_host_perf_mode;
+    const char* rs2_host_perf_mode_to_string( rs2_host_perf_mode perf );
```

Comment:
This should be added into the public API - realsense.def and then added into the wrappers (above)

Reply:
added.